src/lib/features/playground/offline-unleash-client.ts

```diff
@@ -45,9 +45,9 @@ export const mapFeatureForClient = (
             })),
             constraints:
                 strategy.constraints?.map((constraint) => ({
-                    inverted: false,
```

Comment:
I believe this was a bug, it was being overridden by `...constraint` when set. But constraint type could have inverted as undefined, but now that defaults can't be undefined it's always overridden and the compiler tells this is not needed